tsconfig.json

```diff
@@ -12,7 +12,8 @@
       }
     ],
     "paths": {
-      "@/*": ["./src/*", "./styles/*"]
+      "@/*": ["./src/*"],
+      "@penumbra-zone/ui/*": ["./src/shared/ui/*"]
```

Comment:
this allows importing `@penumbra-zone/ui/Component` from anywhere in the DEX code